mw-webapp/src/component/userCard/UserCard.module.scss

```diff
@@ -83,3 +85,11 @@ $userCardAdditionalHeight: 82px;
 .ways {
   flex-wrap: wrap;
 }
+
+.flagMentor {
+  position: absolute;
+  top: 0;
+  right: $flagMenorRightOffset;
+  fill: var(--bookmarksIconColor);
+  stroke: none
```

Comment:
Why stroke is 0?

Reply:
in figma maket we don't have stroke, only fill 
![image](https://github.com/tritonJS826/masters-way/assets/112805668/a77f9654-7278-4736-8c59-91c390387fb9)